src/modules/musig/example.c

```diff
@@ -85,9 +85,9 @@ int sign(const secp256k1_context* ctx, unsigned char seckeys[][32], const secp25
     for (i = 0; i < N_SIGNERS; i++) {
         for (j = 0; j < N_SIGNERS; j++) {
             if (!secp256k1_musig_set_nonce(ctx, &signer_data[i][j], &nonce[j])) {
-                /* Signer j's nonce does not match the nonce commitment. Wait
-                 * until the correct nonce is received or restart the protocol
-                 * (with a different session ID of course). */
+                /* Signer j's nonce does not match the nonce commitment. In this
+                 * case, restart the protocol by creating a new session (with a fresh
+                 * session ID of course). */
```

Comment:
nit: I'd even say "In this case, abort the protocol. It's possible to restart the protocol..."